libmscore/layout.cpp

```diff
@@ -3735,7 +3721,7 @@ System* Score::collectSystem(LayoutContext& lc)
                 } else {
                     m->removeSystemHeader();
                 }
-                while (true) {
+                for (;;) {
                     // TODO: what if the nobreak group takes the entire system - is this correct?
```

Comment:
@MarcSabatella I think the answer is yes, but why might it not be?

Reply:
No idea, but somehow it seems like something to be sure to test.